config/config.exs

```diff
@@ -63,6 +58,16 @@ config :hex_web, HexWeb.Endpoint,
   pubsub: [name: HexWeb.PubSub,
            adapter: Phoenix.PubSub.PG2]
 
+config :hex_web, HexWeb.Mailer,
+  adapter: Bamboo.SMTPAdapter,
+  server: System.get_env("HEX_SES_ENDPOINT") || "email-smtp.us-west-2.amazonaws.com",
+  port: System.get_env("HEX_SES_PORT") || "587",
+  username: System.get_env("HEX_SES_USERNAME"),
+  password: System.get_env("HEX_SES_PASSWORD"),
+  tls: :if_available, # can be `:always` or `:never`
+  ssl: false,
```

Comment:
I am not sure what's the difference, the documentation on this bit is vague. I'll take your suggestion and fix it, need to dive in the code to figure out what's the actual difference.